srcpkgs/packr/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'packr'
+pkgname=packr
+version=2.8.0
+revision=1
+build_style=go
+go_import_path=github.com/gobuffalo/${pkgname}/v2/packr2
+short_desc="Static files embedder for Go binaries"
+maintainer="Gabriel Sanches <gabriel@gsr.dev>"
+license="MIT"
+homepage="https://github.com/gobuffalo/packr"
+distfiles="https://github.com/gobuffalo/${pkgname}/archive/v${version}.tar.gz"
+checksum=2cab1e8d60734af05d594346f8e4bffcef0ac2aea4895b08bd9f3c40fc24c639
+
+pre_install() {
+	mv ${GOPATH}/bin/packr{2,}
```

Comment:
same as above.

Reply:
For this case, I agree, the right thing should be to have `packr2` as proposed from its original authors [here](https://github.com/gobuffalo/packr/blob/master/v2/README.md). Maybe I should name the package `packr2` instead of `packr`?

Reply:
yeah that sounds good.

Reply:
Alright, now it's `packr2`.